src/component.js

```diff
@@ -104,17 +104,17 @@ export function getDomSibling(vnode, childIndex) {
 
 /**
  * Trigger in-place re-rendering of a component.
- * @param {import('./internal').Component} c The component to rerender
+ * @param {import('./internal').Component} component The component to rerender
  */
 function renderComponent(component) {
 	let vnode = component._vnode,
 		oldDom = vnode._dom,
 		parentDom = component._parentDom;
 
 	if (parentDom) {
-		let mounts = [];
-		let newDom = diff(parentDom, vnode, assign({}, vnode), component._context, parentDom.ownerSVGElement!==undefined, null, mounts, oldDom == null ? getDomSibling(vnode) : oldDom);
-		commitRoot(mounts, vnode);
+		let commitQueue = [];
```

Comment:
Do people like this rename of `mounts` to `commitQueue`? Since the `mounts` array isn't just used for `componentDidMount` anymore I figured it probably needed a new name that better communicated that.

Reply:
+1 for the name change. It better reflects the actual thing it's used for 👍